codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/RestJson1.kt

```diff
@@ -315,20 +298,14 @@ class RestJson1HttpSerializerGenerator(
             }
             rust(
                 """let response = #T(&expected).expect("failed to serialize response");""",
-                serde.member(fnName)
+                operation.member(fnName)
```

Comment:
generally, you'll probably want to test the external API instead of testing the serializers directly (but maybe the external API doesn't exist yet?)

That also means you don't need to guess at the `fnName`

Reply:
The external API is not there yet as you rightly guessed. David is working on extractors for Json and once that is done we will start autogenerating the public and more stable API. Little steps 😄